hivemind/p2p/p2p_daemon_bindings/utils.py

```diff
@@ -46,6 +46,7 @@ async def write_unsigned_varint(stream: asyncio.StreamWriter, integer: int, max_
             value |= 0x80
         byte = value.to_bytes(1, "big")
         stream.write(byte)
+        await stream.drain()
```

Comment:
drain() is [recommended](https://docs.python.org/3/library/asyncio-stream.html#asyncio.StreamWriter.write) after write()